src/main/java/appeng/thirdparty/codechicken/lib/model/pipeline/transformers/QuadReInterpolator.java

```diff
@@ -0,0 +1,62 @@
+package appeng.thirdparty.codechicken.lib.model.pipeline.transformers;
+
+import appeng.thirdparty.codechicken.lib.math.InterpHelper;
+import appeng.thirdparty.codechicken.lib.model.CachedFormat;
+import appeng.thirdparty.codechicken.lib.model.Quad;
+import appeng.thirdparty.codechicken.lib.model.Quad.Vertex;
+import appeng.thirdparty.codechicken.lib.model.pipeline.IPipelineElementFactory;
+import appeng.thirdparty.codechicken.lib.model.pipeline.QuadTransformer;
+
+/**
+ * This transformer Re-Interpolates the Color, UV's and LightMaps.
+ * Use this after all transformations that translate vertices in the pipeline.
+ *
+ * This Transformation can only be used in the BakedPipeline.
+ *
+ * Created by covers1624 on 30/07/18.
+ */
+public class QuadReInterpolator extends QuadTransformer {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)